src/NzbDrone.Api/Config/DownloadClientConfigModule.cs

```diff
@@ -18,7 +18,7 @@ public class DownloadClientConfigModule : NzbDroneConfigModule<DownloadClientCon
                            .SetValidator(rootFolderValidator)
                            .SetValidator(mappedNetworkDriveValidator)
                            .SetValidator(pathExistsValidator)
-                           .When(c => !string.IsNullOrWhiteSpace(c.DownloadedEpisodesFolder));
+                           .When(c => string.IsNotNullOrWhiteSpace(c.DownloadedEpisodesFolder));
```

Comment:
Same here, not a method on `string`, should be converted to `c.DownloadedEpisodesFolder.IsNotNullOrWhiteSpace()`